src/features/world/ui/NPCModals.tsx

```diff
@@ -110,15 +86,6 @@ export const NPCModals: React.FC<Props> = ({ onClose, onOpen }) => {
             </div>
           </CloseButtonPanel>
         )}
-        {npc === "igor" && (
```

Comment:
Any thoughts on how we can access the Blacksmith/Crafting modal now?

It would be nice if you could choose to "Craft" or "Do delivery"